content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -90,54 +91,62 @@ and debug issues. The compromise, however, is that you must start with knowledge
-however, the Kubelet is still bootstrapped with command-line flags. Fortunately,
-there is a dirty trick you can use to generate a config file containing a Node's
-current configuration. The trick involves accessing the Kubelet server's `configz`
-endpoint via the kubectl proxy. This endpoint, in its current implementation, is
-intended to be used only as a debugging aid, which is part of why this is a
-dirty trick. The endpoint may be improved in the future, but until then
-it should not be relied on for production scenarios.
-This trick also requires the `jq` command to be installed on your machine,
-for unpacking and editing the JSON response from the endpoint.
-
-Do the following to generate the file:
-
-1. Pick a Node to reconfigure. We will refer to this Node's name as NODE_NAME.
-2. Start the kubectl proxy in the background with `kubectl proxy --port=8001 &`
-3. Run the following command to download and unpack the configuration from the
-configz endpoint:
-
-```
-$ export NODE_NAME=the-name-of-the-node-you-are-reconfiguring
-$ curl -sSL http://localhost:8001/api/v1/nodes/${NODE_NAME}/proxy/configz | jq '.kubeletconfig|.kind="KubeletConfiguration"|.apiVersion="kubelet.config.k8s.io/v1beta1"' > kubelet_configz_${NODE_NAME}
-```
-
-Note that we have to manually add the `kind` and `apiVersion` to the downloaded
-object, as these are not reported by the configz endpoint. This is one of the
-limitations of the endpoint.
-
-### Edit the configuration file
-
-Using your editor of choice, change one of the parameters in the
-`kubelet_configz_${NODE_NAME}` file from the previous step. A QPS parameter,
-`eventRecordQPS` for example, is a good candidate.
-
-### Push the configuration file to the control plane
+Currently, the Kubelet is bootstrapped with **a combination of this file and command-line flags**
+that can override the configuration in the file.
+As a workaround, you can generate a config file containing a Node's current
+configuration by accessing the Kubelet server's `configz` endpoint via the
+kubectl proxy. This endpoint, in its current implementation, is intended to be
+used only as a debugging aid. Do not rely on the behavior of this endpoint for
+production scenarios. The examples below use the `jq` command to streamline
+working with JSON. To follow the tasks as written, you need to have `jq`
+installed, but you can adapt the tasks if you prefer to extract the
+`kubeletconfig` subobject manually.
+
+#### Generate the configuration file
+
+1.  Choose a Node to reconfigure. In this example, the name of this Node is
+    referred to as `NODE_NAME`.
+2.  Start the kubectl proxy in the background using the following command: 
+      ```bash
+      kubectl proxy --port=8001 &
+      ```
+3.  Run the following command to download and unpack the configuration from the
+    `configz` endpoint. The command is long, so be careful when copying and
+    pasting. **If you use zsh**, replace the `${NODE_NAME}` in the URL with the
```

Comment:
You can still use the variable, but you have to edit out the backslashes that get inserted during the paste before running the command.
1. Copy `https://www.example.com/${NODE_NAME}`
2. When pasted, it turns into this: `https://www.example.com/$\{NODE_NAME\}`
3. So it needs to be edited back to this: `https://www.example.com/${NODE_NAME}`

Reply:
Otherwise all of this looks good! Thanks for your help!
I'm removing the hold anyway, because we can clarify this later if necessary/

Reply:
#9061